build/test-requirements.txt

```diff
@@ -1,5 +1,5 @@
 flake8
-jaxlib==0.1.51
+jaxlib==0.1.55
```

Comment:
You should also bump the minimum jaxlib here: https://github.com/google/jax/blob/master/jax/lib/__init__.py#L25
(In fact, can you also add a comment to the line here saying to always keep these versions in sync?)

Reply:
Done, thank you!